general-superstaq/general_superstaq/user_config.py

```diff
@@ -96,6 +98,9 @@ def ibmq_set_token(self, token: str) -> str:
         Returns:
             String containing status of update (whether or not it failed).
         """
+        public_key = rsa.key.PublicKey(int(gss.TOKEN_PUBLIC_KEY_N), gss.TOKEN_PUBLIC_KEY_E)
+        token = rsa.encrypt(token.encode(), public_key)
+        token = str(rsa.transform.bytes2int(token))
```

Comment:
why not just use `_bytes_to_str` for this? (and `_str_to_bytes` server-side)

Reply:
> why not just use _bytes_to_str for this? (and _str_to_bytes server-side)

We didn't know about this.

Reply:
> We didn't know about this.

haha yeah i'd totally forgotten about it too until like an hour ago when i ran into it working on something unrelated

Reply:
one other thought - would it make sense to move this functionality to `gss.serialization` as well? e.g.
```python
def encrypt(val: str) -> str:
    public_key = ...
    encrypted_val = rsa.encrypt(val.encode(), public_key)
    return bytes_to_str(encrypted_val)
```
so we could just call it here and also use it elsewhere if we want to start encrypting other tokens/values/etc

Reply:
I like that idea. Let's keep in our pockets for when [we will start encrypting other tokens/values ](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))